core/src/com/unciv/logic/simulation/Simulation.kt

```diff
@@ -92,6 +101,16 @@ class Simulation(
         println("Simulation step ($stepCounter/$maxSimulations)")
     }
 
+    @Synchronized
+    fun saveStat(gameInfo: GameInfo) {
+        for (civ in gameInfo.civilizations.filter { it.civName != Constants.spectator }) {
+            var popsum = 0
+            civ.cities.forEach { city -> popsum += city.population.population }
+            //println("$civ $popsum")
+            avgStat[civ.civName]!!.set(avgStat[civ.civName]!!.get() + popsum)
```

Comment:
Since this saves the sum and not the average, please call it an appropriate babe

Reply:
Fixed